apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Snippets/SpannerConnectionSnippets.cs

```diff
@@ -36,5 +58,251 @@ public void CreateConnection()
             Assert.Equal("my-instance", connection.SpannerInstance);
             Assert.Equal("my-db", connection.Database);
         }
+
+        [Fact]
+        public async Task CreateDatabaseAsync()
+        {
+            // CreateDatabaseAsync will create a new database and dispose of it right after.
+            // But for consistency sake, we want to show the same variables in the snippets.
+            _databaseName = "t_" + Guid.NewGuid().ToString("N").Substring(0, 28);
+
+            // Sample: CreateDatabaseAsync
+            // Additional: CreateDdlCommand
+            using (var connection =
+                new SpannerConnection($"Data Source=projects/{_projectId}/instances/{_instanceName}/databases/{_databaseName}"))
+            {
+                var createDbCmd = connection.CreateDdlCommand($"CREATE DATABASE {_databaseName}");
+                await createDbCmd.ExecuteNonQueryAsync();
+
+                var createTableCmd = connection.CreateDdlCommand(
+                    @"CREATE TABLE TestTable (
+                                            Key                STRING(MAX) NOT NULL,
+                                            StringValue        STRING(MAX),
+                                            Int64Value         INT64,
+                                          ) PRIMARY KEY (Key)");
+                await createTableCmd.ExecuteNonQueryAsync();
+            }
+            // End sample
+
+            using (var connection =
+                new SpannerConnection($"Data Source=projects/{_projectId}/instances/{_instanceName}"))
+            {
+                var createDbCmd = connection.CreateDdlCommand($"DROP DATABASE {_databaseName}");
+                await createDbCmd.ExecuteNonQueryAsync();
+            }
+        }
+
+        [Fact]
+        public async Task InsertDataAsync()
+        {
+            await _fixture.EnsureTestDatabaseAsync().ConfigureAwait(false);
+
+            // Sample: InsertDataAsync
+            using (var connection = new SpannerConnection(
+                $"Data Source=projects/{_projectId}/instances/{_instanceName}/databases/{_databaseName}"))
+            {
+                await connection.OpenAsync();
+
+                var cmd = connection.CreateInsertCommand(
+                    "TestTable", new SpannerParameterCollection
+                    {
+                        {"Key", SpannerDbType.String},
+                        {"StringValue", SpannerDbType.String},
+                        {"Int64Value", SpannerDbType.Int64}
+                    });
+
+                // This executes 5 distinct transactions with one row written per transaction.
+                for (var i = 0; i < 5; i++)
+                {
+                    cmd.Parameters["Key"].Value = Guid.NewGuid().ToString("N");
+                    cmd.Parameters["StringValue"].Value = $"StringValue{i}";
+                    cmd.Parameters["Int64Value"].Value = i;
+                    await cmd.ExecuteNonQueryAsync();
```

Comment:
Print out this result, and maybe comment that it's 1?

Reply:
seems reasonable here.